### PR TITLE
butane: renamed from fcct

### PR DESCRIPTION
**Pull request by Okeanos**
Renamed the formula based on the official rename (see coreos/butane/issues/167 ) as discussed in the version bump pull request (see Homebrew/homebrew-core/pull/75077 ).

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/butane.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-mod=vendor",
+      *std_go_args(ldflags: "-w -X github.com/coreos/butane/internal/version.Raw=#{version}"), "internal/main.go"
+
+    prefix.install_metafiles
```
Comment (carlocab) [inline Formula/butane.rb:20]:
```suggestion
```
This is no longer needed.

**Review by dawidd6 — DISMISSED** [review]

**Comment by carlocab** [comment]
@Okeanos, can you squash your commits together? BrewTestBot can't merge this otherwise.

**Comment by carlocab** [comment]
@dawidd6 would've been nicer to squash those together -- keeps the log clean.

**Comment by dawidd6** [comment]
Heh. No need for that. Still it could have been squashed...

**Comment by Okeanos** [comment]
Yeah, I didn't know what was preferred and because the review remark was conveniently marked as "fix in inline by clicking this magic button" in GitHub I just did that. Sorry 😅 .

**Comment by carlocab** [comment]
Yea, in the future please keep it to one-commit-per-formula-file-modified (and one-formula-file-modified-per-commit). Thanks for your PR!

**Pull request closed without merging**